quickstart/python/PythonApplication1.py

```diff
@@ -16,6 +16,6 @@
 print("GET Message returned : " + result.decode("utf-8"))
 
 result = r.client_list()
-print("CLIENT LIST returned : ")
+print(f"CLIENT LIST returned : ")
```

Comment:
I wonder if the spirit of suggestion from the Python developer was to use this `f""` [formatted string](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) syntax instead of `+` string concatenation in all of the print statements that currently use `+` string concatenation.

The tiny irony would be that this exact line at line 19 then wouldn't need to change because it's just a single string with no concatenation.

Reply:
@brendanzagaeski - I can't believe how long it has taken me to get back to this. Do you think `f""` syntax should be applied to all the strings by custom?

I just want this to get done at this point.